string roomname
---
geometry_msgs/Point point
